conf/generator.yaml

```diff
@@ -145,3 +160,102 @@ modules:
       - artifactid: spring-boot-starter-test
         groupid: org.springframework.boot
         scope: test
+  - name: tracing
+    description: "Distributed tracing with OpenTracing"
+    guide_ref: ""
+    tags:
+      - tracing
+      - monitoring
+    dependencies:
+      - artifactid: opentracing-spring-jaeger-web-starter
+        groupid: io.opentracing.contrib
+      - artifactid: spring-boot-starter
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: infinispan
+    description: "Remote Infinispan cache and datagrid"
+    guide_ref: ""
+    tags:
+      - persistence
+      - cache
+    dependencies:
+      - artifactid: infinispan-spring-boot-starter-remote
+        groupid: org.infinispan
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: infinispan-embedded
+    description: "Embedded Infinispan cache and datagrid"
+    guide_ref: ""
+    tags:
+      - persistence
+      - cache
+    dependencies:
+      - artifactid: infinispan-spring-boot-starter-embedded
+        groupid: org.infinispan
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: artemis
+    description: "Artemis message broker"
+    guide_ref: ""
+    tags:
+      - messaging
+      - jms
+    dependencies:
+      - artifactid: spring-boot-starter-artemis
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: activemq
+    description: "ActiveMQ message broker"
+    guide_ref: ""
+    tags:
+      - messaging
+      - jms
```

Comment:
As Artemis supports also the protocols `A-MQP, MQTT,STOMP`, then they should be mentioned like also for the versions `1.1 and 2.0 support`

Reply:
I thought that it might be a bit misleading, because JMS is automatically configured by the starter and no client for other protocol is provided.

Reply:
Good to know. I wasn't aware that JMS's protocol was the default (= artifact -> artemis-jms-client) and to use A-MQP | STOMP | MQTT ..., then another client is needed. So, I agree with you that including them could be confusing